### DISCUSSION TITLE
Omitting columns from the sync process

### DISCUSSION THREAD

**Discussion by henda79** [Azure/azure-mobile-apps#680] [Q&A]
I have some tables which have columns and are really only for the client. Is there a way to omit these from the sync ? I thought about some of the annotations I could use on the properties but ones like `JsonIgnore` would stop the column been created on the client wouldn't it ?!?

What's the best way to go about this if any ?

Also, is it possible to have table(s) in the database which does not sync at all. It seems overkill to have another database and all the fluff that comes with it for the sake of 1 or 2 tables.

Thanks !

**Comment by adrianhall** [comment]
There isn't really a good way to do this right now.  However, I feel you have a good point, so I'm swapping this to an issue.

The correct way to do this right now is a [JsonIgnore] attribute, I believe, but I want to check it out and perhaps add a few tests for it.
